checks/documentation.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package checks
+
+import (
+	//nolint
+	_ "embed"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/pkg/errors"
+)
+
+//go:embed checks.yaml
+var checks string
+
+type documentation struct {
+	Checks struct {
+		SecurityPolicy struct {
```

Comment:
This is a lot of repetition and makes maintenance difficult. Can we marshal these into a `map[string]CheckDescription` or something similar?

Reply:
There is also a con with that approach, we lose the type-safety and strings are all over the codebase. 

We would have to define a constant for the string and use it which beats the purpose.